### PR TITLE
{bio}[gompi/2019b] Kaiju v1.7.3 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node377.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/62e6e50361cdb7e9c90ec96ac35628c2 for a full test report.

File: easybuild/easyconfigs/k/Kaiju/Kaiju-1.7.3-gompi-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+sanity_check_paths = {
+    'files': [
+        'bin/%s' % x for x in [
+            'kaiju', 'kaiju2krona', 'kaiju2table', 'kaiju-addTaxonNames',
+            'kaiju-convertMAR.py', 'kaiju-convertNR', 'kaiju-gbk2faa.pl',
+            'kaiju-makedb', 'kaiju-mergeOutputs', 'kaiju-mkbwt', 'kaiju-mkfmi',
+            'kaijup', 'kaijux'
+        ]
+    ],
```
Comment (lexming) [inline easybuild/easyconfigs/k/Kaiju/Kaiju-1.7.3-gompi-2019b-Python-3.7.4.eb]: This part could be nicely compressed as follows
```
    'files': ['bin/%s' % x for x in ['kaiju', 'kaiju2krona', 'kaiju2table', 'kaiju-addTaxonNames',
                                     'kaiju-convertMAR.py', 'kaiju-convertNR', 'kaiju-gbk2faa.pl', 'kaiju-makedb',
                                     'kaiju-mergeOutputs', 'kaiju-mkbwt', 'kaiju-mkfmi', 'kaijup', 'kaijux']],
```

Comment (smoors) [reply easybuild/easyconfigs/k/Kaiju/Kaiju-1.7.3-gompi-2019b-Python-3.7.4.eb]: ok, we can compress even more :)

Comment (lexming) [reply easybuild/easyconfigs/k/Kaiju/Kaiju-1.7.3-gompi-2019b-Python-3.7.4.eb]: nice :+1:

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7547cfcccfed48567d2db04054488e64 for a full test report.

**Comment by lexming** [comment]
LGTM

**Pull request merged**